understanding/change-on-request.html

```diff
@@ -58,87 +56,67 @@
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 3.2.5:</span>Change on Request (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 3.2.5:</span>要求による変化 (レベル AAA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
                   <p>
-                     <a href="#dfn-changes-of-context">Changes of context</a> are initiated only by user request or a <a href="#dfn-mechanism">mechanism</a> is available to turn off such changes.
+                     <a href="#dfn-changes-of-context">コンテキストの変化</a>は利用者の要求によってだけ生じるか、又は、そのような変化を止める<a href="#dfn-mechanism">メカニズム</a>が利用できる。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   		         
-                  <dt>Goal</dt>
-                  <dd>Users have full control of major content changes.</dd>
+                  <dt>目標</dt>
+                  <dd>利用者は、主要なコンテンツの変更を完全に制御できる。</dd>
                   		         
-                  <dt>What to do</dt>
-                  <dd>Provide ways for users to trigger or turn off changes of context.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>利用者がコンテキストの変化を引き起こす又はオフにする方法を提供する。</dd>
```

Comment:
「又は」の前後でどこがどこにかかっているのかわかりにくく、読みにくい感じがします。まあ原文もそうなんですが。あとやはり trigger がわかりにくいですね。「自身で引き起こす」くらいに補った方が良いのかもしれません。

あとは全体として、Provide ways for users なので「利用者に…」とした方が良いのかもしれません。たとえばこうしてみてはどうでしょうか。

```suggestion
                  <dd>利用者に対して、コンテキストの変化を自身で引き起こす、又はオフにする方法を提供する。</dd>
```